kedify-agent/values.yaml

```diff
@@ -0,0 +1,85 @@
+agent:
+  replicas: 1
+
+  # you should already have this
+  orgId: ""
+  # you should already have this
+  apiKey: ""
+# agentId: abcd1234-ab34-..
+  kedifyServer: api.kedify.io
```

Comment:
This URL serves the frontentd.

Proxy address for prod is `service.kedify.io:443`